[2025-09-24T18:44:13.383+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_postgres_dag.select_users manual__2025-09-24T18:44:07.960057+00:00 [queued]>
[2025-09-24T18:44:13.389+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_postgres_dag.select_users manual__2025-09-24T18:44:07.960057+00:00 [queued]>
[2025-09-24T18:44:13.389+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-09-24T18:44:13.398+0000] {taskinstance.py:2191} INFO - Executing <Task(PostgresOperator): select_users> on 2025-09-24 18:44:07.960057+00:00
[2025-09-24T18:44:13.403+0000] {standard_task_runner.py:60} INFO - Started process 100 to run task
[2025-09-24T18:44:13.405+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'test_postgres_dag', 'select_users', 'manual__2025-09-24T18:44:07.960057+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmp5bufna1k']
[2025-09-24T18:44:13.407+0000] {standard_task_runner.py:88} INFO - Job 11: Subtask select_users
[2025-09-24T18:44:13.441+0000] {task_command.py:423} INFO - Running <TaskInstance: test_postgres_dag.select_users manual__2025-09-24T18:44:07.960057+00:00 [running]> on host d59ad7cc807c
[2025-09-24T18:44:13.493+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_postgres_dag' AIRFLOW_CTX_TASK_ID='select_users' AIRFLOW_CTX_EXECUTION_DATE='2025-09-24T18:44:07.960057+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-24T18:44:07.960057+00:00'
[2025-09-24T18:44:13.496+0000] {sql.py:276} INFO - Executing: SELECT * FROM public.users;
[2025-09-24T18:44:13.502+0000] {base.py:83} INFO - Using connection ID 'postgres' for task execution.
[2025-09-24T18:44:13.510+0000] {base.py:83} INFO - Using connection ID 'postgres' for task execution.
[2025-09-24T18:44:13.513+0000] {sql.py:450} INFO - Running statement: SELECT * FROM public.users;, parameters: None
[2025-09-24T18:44:13.514+0000] {sql.py:459} INFO - Rows affected: 1
[2025-09-24T18:44:13.529+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=test_postgres_dag, task_id=select_users, execution_date=20250924T184407, start_date=20250924T184413, end_date=20250924T184413
[2025-09-24T18:44:13.578+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-24T18:44:13.589+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
